cppcache/src/ExecutionImpl.cpp

```diff
@@ -429,33 +429,20 @@ void ExecutionImpl::executeOnAllServers(const std::string& func,
   std::shared_ptr<CacheableString> exceptionPtr = nullptr;
   GfErrType err = tcrdm->sendRequestToAllServers(
       func.c_str(), getResult, timeout, m_args, m_rc, exceptionPtr);
-  if (exceptionPtr != nullptr && err != GF_NOERR) {
-    LOGDEBUG("Execute errorred: %d", err);
-    // throw FunctionExecutionException( "Execute: failed to execute function
-    // with server." );
-    if (err == GF_CACHESERVER_EXCEPTION) {
-      throw FunctionExecutionException(
-          "Execute: failed to execute function with server.");
-    } else {
-      throwExceptionIfError("Execute", err);
-    }
-  }
-
-  if (err == GF_AUTHENTICATION_FAILED_EXCEPTION ||
-      err == GF_NOT_AUTHORIZED_EXCEPTION ||
-      err == GF_AUTHENTICATION_REQUIRED_EXCEPTION) {
-    throwExceptionIfError("Execute", err);
-  }
 
   if (err != GF_NOERR) {
+    LOGDEBUG("Execute errorred: %d", err);
```

Comment:
Sorry to be "that guy" about grammar, but errorred isn't really a word.  Should be "erred" or just avoid it altogether and use "failed".  Thx

Reply:
Ugh, and it was even like this to begin with - yikes, sorry.

Reply:
No problem. It looked weird to me too. I even thought the author might want to say something like "error red" :-).